### DISCUSSION TITLE
Reloading Without Restarting

### DISCUSSION THREAD

**Discussion by EwyBoy** [EwyBoy/OreTweaker#68] [Ideas]
Being able to reload / rebuild the ore registries without having to hard restart the client / server each time you make a change to any json files. This feature might not be possible but I am willing to spend some R&D time on this issue to improve the user experience of the mod if this is a feature enough people may want.

Please comment any input you may have on this feature request below.

**Comment by Goldenlion5648** [comment]
Yes please! This would speed up development so much!
